controllers/base_controller.js

```diff
@@ -329,7 +329,6 @@ module.exports = function BaseControllerModule(pb) {
                 return;
```

Comment:
Remember Allegis wants an alternate link for the current language. This code will skip it.